app/static/js/events/wizard/sessions-speakers/vue-models.js

```diff
@@ -1,3 +1,16 @@
+Date.prototype.ddmmyyyy = function() {
```

Comment:
This is not overriding any preexisting methods of the said native object. So IMO it's okay to let this go.